modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -85,17 +67,7 @@ extension Date: JSONEncodable {
     }
 }
 
-extension URL: JSONEncodable {
-    func encodeToJSON() -> Any {
-        return self
-    }
-}
-
-extension UUID: JSONEncodable {
-    func encodeToJSON() -> Any {
-        return self.uuidString
-    }
-}{{/useVapor}}{{#generateModelAdditionalProperties}}
+{{/useVapor}}{{#generateModelAdditionalProperties}}
```

Comment:
@funzin can you please make this one line only to avoid generate extra new lines? Thanks
```
}{{/useVapor}}{{#generateModelAdditionalProperties}}
```

Reply:
I unserstand:+1:
Remove empty lines
e69c46f